lib/hackney/income/stored_tenancies_gateway.rb

```diff
@@ -82,6 +82,8 @@ def tenancies_filtered_for(user_id, filters)
           end
         end
 
+        query = query.where('eviction_date >= ?', Date.today).order('eviction_date') if filters[:upcoming_evictions].present?
```

Comment:
- `Date.today` is not Time zone aware. 
- From the `schema.rb`: `t.datetime "eviction_date"` so we should query a Date Time object


```suggestion
        query = query.where('eviction_date >= ?', Time.zone.now.beginning_of_day).order('eviction_date') if filters[:upcoming_evictions].present?
```